src/commands/SetTrackInfoCommand.cpp

```diff
@@ -286,17 +286,20 @@ static const EnumValueSymbol kColourStrings[nColours] =
 
 enum kScaleTypes
 {
-   kLinear,
-   kDb,
+   kLinearAmp,
+   kLogarithmicDb,
+   kLinearDb,
    nScaleTypes
 };
 
 static const EnumValueSymbol kScaleTypeStrings[nScaleTypes] =
 {
    // These are acceptable dual purpose internal/visible names
-   { XO("Linear") },
+   { XO("Linear (amp)") },
```

Comment:
This changed line should be preceded by an `i18n-hint` comment explaining that "amp" abbreviates "amplitude".

Those comments matter because `xgettext` extracts them to `locale/audacity.pot` when we update that file, at least once in each development cycle.  Then the comments are copied to the several `locale/*.po` files, and serve as clues for the volunteer translators who update the string catalogs for other languages.

Reply:
See `scripts/update_po_files.sh` if you are curious about that process.